CRITICAL:root:{'date': datetime.datetime(2017, 10, 2, 0, 45, 27, 311953), 'type': 'critical', 'error': '!! Traceback (most recent call last):\n!!   File "/Users/victorg/Documents/ACADEMY/code/code2/webfunctions.py", line 47, in __request\n    response = urllib.request.urlopen(self.__request)\n!!   File "/usr/local/Cellar/python3/3.6.2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/urllib/request.py", line 223, in urlopen\n    return opener.open(url, data, timeout)\n!!   File "/usr/local/Cellar/python3/3.6.2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/urllib/request.py", line 517, in open\n    req.timeout = timeout\n!! AttributeError: \'method\' object has no attribute \'timeout\'\n'}
CRITICAL:root:{'date': datetime.datetime(2017, 10, 2, 0, 51, 4, 624750), 'type': 'critical', 'error': '!! Traceback (most recent call last):\n!!   File "/Users/victorg/Documents/ACADEMY/code/code2/webfunctions.py", line 50, in __request\n    response = urllib.request.urlopen(self.__request)\n!!   File "/usr/local/Cellar/python3/3.6.2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/urllib/request.py", line 223, in urlopen\n    return opener.open(url, data, timeout)\n!!   File "/usr/local/Cellar/python3/3.6.2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/urllib/request.py", line 517, in open\n    req.timeout = timeout\n!! AttributeError: \'method\' object has no attribute \'timeout\'\n'}
INFO:root:{'date': datetime.datetime(2017, 10, 2, 0, 52, 22, 415055), 'type': 'info', 'info': 'request', 'event': 'url request successfull'}
DEBUG:root:{'date': datetime.datetime(2017, 10, 2, 0, 52, 22, 415305), 'type': 'debug', 'info': 'http://localhost:8888/agenda'}
INFO:root:{'date': datetime.datetime(2017, 10, 2, 0, 52, 40, 207679), 'type': 'info', 'info': 'request', 'event': 'url request successfull'}
DEBUG:root:{'date': datetime.datetime(2017, 10, 2, 0, 52, 40, 207949), 'type': 'debug', 'info': 'http://localhost:8888/agenda'}
INFO:root:{'date': datetime.datetime(2017, 10, 2, 0, 52, 40, 681024), 'type': 'info', 'info': 'request', 'event': 'url request successfull'}
DEBUG:root:{'date': datetime.datetime(2017, 10, 2, 0, 52, 40, 681185), 'type': 'debug', 'info': 'http://localhost:8888/agenda'}
CRITICAL:root:{'date': datetime.datetime(2017, 10, 2, 0, 55, 47, 468174), 'type': 'critical', 'error': '!! Traceback (most recent call last):\n!!   File "/Users/victorg/Documents/ACADEMY/code/code2/webfunctions.py", line 50, in __request\n    response = urllib.request.urlopen(req,action)\n!! NameError: name \'action\' is not defined\n'}
INFO:root:{'date': datetime.datetime(2017, 10, 2, 0, 56, 29, 814691), 'type': 'info', 'info': 'request', 'event': 'url request successfull'}
DEBUG:root:{'date': datetime.datetime(2017, 10, 2, 0, 56, 29, 814946), 'type': 'debug', 'info': 'http://localhost:8888/agenda/api/login_json.php'}
INFO:root:{'date': datetime.datetime(2017, 10, 2, 0, 56, 30, 314627), 'type': 'info', 'info': 'request', 'event': 'url request successfull'}
DEBUG:root:{'date': datetime.datetime(2017, 10, 2, 0, 56, 30, 314796), 'type': 'debug', 'info': 'http://localhost:8888/agenda/api/select_json.php'}
